smarty/templates/email/issue_change.tpl

```diff
@@ -1,4 +1,4 @@
-Subject: Change to Issue # - {$issueID} [{$study}]
+Subject: Change to Issue - #{$issueID} [{$study}]
```

Comment:
This is really specific, but there's an extra line in this email when no comment is given: 
<img width="710" height="375" alt="image" src="https://github.com/user-attachments/assets/bcccf9bc-fdf0-4a70-a6c4-76f99c18e980" />